Gemfile

```diff
@@ -1,10 +1,12 @@
 source 'https://rubygems.org'
 
-# gem 'fhir_models', :path => '../fhir_models'
-# gem 'fhir_dstu2_models', :path => '../fhir_dstu2_models'
-
 gemspec
 
+gem 'fhir_models', git: 'https://github.com/fhir-crucible/fhir_models.git', branch: 'master'
+gem 'fhir_stu3_models', git: 'https://github.com/fhir-crucible/fhir_stu3_models.git', branch: 'master'
+gem 'fhir_dstu2_models', git: 'https://github.com/fhir-crucible/fhir_dstu2_models.git', branch: 'update_dep'
```

Comment:
why aren't you using the `master` branch of `fhir_dstu2_models`?

Reply:
Also, why did you take them out of the gemspec file?

Reply:
There was a dependency conflict between `fhir_dstu2_models` and `fhir_stu3` models

`fhir_dstu2_models` required mime-types (>= 1.16, < 3) and
`fhir_stu3_models` required `mime-types` >=3

The new branch was just to update `mime-types`

Reply:
It is my understanding that the RubyGems gemspec file does not support gems from git repositiories, but Bundler does within the gemfile.  I could totally be wrong about that and would be happy to move them back if there is another way. My intention is to use the gemspec files for these dependencies once updated gems are released.

[Bundler Git Support](https://bundler.io/guides/git.html)
[Specification Reference on Adding Dependencies](https://guides.rubygems.org/specification-reference/#add_runtime_dependency)

Reply:
OK, I understand this is a WIP, I just expect these to to moved back into gemspec before we merge this to master... meaning any upstream dependencies will need to be merged first.

Reply:
Moved them back into the gemspec